Tests/Test.Eyes.Selenium.DotNet/Resources/ExcludedTests.txt

```diff
@@ -10,6 +10,9 @@ Applitools.Selenium.Tests.TestSpecialCases(Chrome, Scroll).TestCheckRegionInAVer
 Applitools.Selenium.Tests.TestSpecialCases(Chrome, Scroll).TestCheckRegionInAVeryBigFrameAfterManualSwitchToFrame
 Applitools.Selenium.Tests.TestSpecialCases(IE, Scroll).TestCheckRegionInAVeryBigFrame
 Applitools.Selenium.Tests.TestSpecialCases(IE, Scroll).TestCheckRegionInAVeryBigFrameAfterManualSwitchToFrame
+Applitools.Selenium.Tests.TestClassicApi(Chrome, CSS).TestCheckWindow
+Applitools.Selenium.Tests.TestClassicApi(Chrome, Scroll).TestCheckWindow
+Applitools.Selenium.Tests.TestClassicApi(Chrome, VG).TestCheckWindow
```

Comment:
Changes to the `ExcludeTests.txt` file are irrelevant as this file isn't in use. Actually, I'd rather delete it.

Reply:
Do this in new PR https://github.com/applitools/Eyes.Sdk.DotNet/pull/16  . Please commit it, this PR can be closed.